### PR TITLE
feat(ui): base layout for my Component Library App

### PR DESCRIPTION
**Pull request by tushartiwari7**

### CODE REVIEW

**Comment by tushartiwari7** [comment]
preview PR: https://elevate-ui-el4ypalep-tushartiwari7.vercel.app/

File: helpers/grid.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.grid {
```
Comment (Priya-Patel31) [inline helpers/grid.css]: Don't make different css files for these utility classes just put them in one file !!

File: helpers/flex.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.flex {
```
Comment (Priya-Patel31) [inline helpers/flex.css]: Don't make different css files for these utility classes just put them in one file !!

**Review by Priya-Patel31 — CHANGES_REQUESTED** [review]

File: helpers/flex.css (hunk 2 above)
Comment (tushartiwari7) [reply helpers/flex.css]: got it, thanks @Priya-Patel31

File: helpers/grid.css (hunk 1 above)
Comment (tushartiwari7) [reply helpers/grid.css]: Implemented [link](https://github.com/tushartiwari7/elevate-ui/pull/2/commits/a73e93dd139013d4aa7485a63845f54da4e5da61)

**Review by Priya-Patel31 — APPROVED** [review]
Changes from your side have made been now you can merge it !!

**Pull request merged**